testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.min_max_03/min.max.jst

```diff
@@ -0,0 +1,29 @@
+JSIGHT 0.3
```

Comment:
Please, add the same tests with `decimal`.

Reply:
fixed https://github.com/jsightapi/jsight-api-core/pull/15/commits/dfa26beb5fdb746976e21a45b6a4be0e43f517a8